src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -269,6 +270,10 @@ private bool IsSetPropertyAccessorNameConflict(SyntaxToken token)
                 => token.IsKind(SyntaxKind.SetKeyword)
                 && IsNameConflictWithProperty("set", token.Parent as AccessorDeclarationSyntax);
 
+            private bool IsInitPropertyAccessorNameConflict(SyntaxToken token)
+                => token.IsKind(SyntaxKind.InitKeyword)
+                && IsNameConflictWithProperty("init", token.Parent as AccessorDeclarationSyntax);
```

Comment:
add test plz.

Reply:
@CyrusNajmabadi I'll add one now.
After careful review, I believe I have a mistake here and the test should fail.
I'll confirm that first then fix the implementation here, which probably should be:

```suggestion
                && IsNameConflictWithProperty("set", token.Parent as AccessorDeclarationSyntax);
```